[2024-07-10 00:21:35,444] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: create_layers_in_snowflake.create_seeds manual__2024-07-10T00:21:34.430321+00:00 [queued]>
[2024-07-10 00:21:35,454] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: create_layers_in_snowflake.create_seeds manual__2024-07-10T00:21:34.430321+00:00 [queued]>
[2024-07-10 00:21:35,454] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-07-10 00:21:35,455] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-07-10 00:21:35,455] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-07-10 00:21:35,469] {taskinstance.py:1377} INFO - Executing <Task(DockerOperator): create_seeds> on 2024-07-10 00:21:34.430321+00:00
[2024-07-10 00:21:35,473] {standard_task_runner.py:52} INFO - Started process 2867 to run task
[2024-07-10 00:21:35,476] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'create_layers_in_snowflake', 'create_seeds', 'manual__2024-07-10T00:21:34.430321+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/create_layers.py', '--cfg-path', '/tmp/tmp8ssyxwpk', '--error-file', '/tmp/tmpykoibtod']
[2024-07-10 00:21:35,477] {standard_task_runner.py:80} INFO - Job 33: Subtask create_seeds
[2024-07-10 00:21:35,534] {task_command.py:369} INFO - Running <TaskInstance: create_layers_in_snowflake.create_seeds manual__2024-07-10T00:21:34.430321+00:00 [running]> on host a22086a40575
[2024-07-10 00:21:35,612] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=create_layers_in_snowflake
AIRFLOW_CTX_TASK_ID=create_seeds
AIRFLOW_CTX_EXECUTION_DATE=2024-07-10T00:21:34.430321+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-07-10T00:21:34.430321+00:00
[2024-07-10 00:21:35,633] {docker.py:247} INFO - Starting docker container from image custom_dbt_image
[2024-07-10 00:21:35,754] {docker.py:308} INFO - 1
[2024-07-10 00:21:35,755] {docker.py:308} INFO - requirements.txt
[2024-07-10 00:21:35,756] {docker.py:308} INFO - 2
[2024-07-10 00:21:35,757] {docker.py:308} INFO - app
bin
games
include
lib
local
sbin
share
src
[2024-07-10 00:21:35,757] {docker.py:308} INFO - 3
[2024-07-10 00:21:35,758] {docker.py:308} INFO - bin
boot
dev
etc
home
lib
lib64
media
mnt
opt
proc
root
run
sbin
srv
sys
tmp
usr
var
[2024-07-10 00:21:35,759] {docker.py:308} INFO - 4
[2024-07-10 00:21:35,759] {docker.py:308} INFO - bin
boot
dev
etc
home
lib
lib64
media
mnt
opt
proc
root
run
sbin
srv
sys
tmp
usr
var
[2024-07-10 00:21:35,825] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=create_layers_in_snowflake, task_id=create_seeds, execution_date=20240710T002134, start_date=20240710T002135, end_date=20240710T002135
[2024-07-10 00:21:35,861] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-07-10 00:21:35,891] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
